R/cpp11.R

```diff
@@ -40,3 +42,16 @@ check_cpp_register_deps <- function() {
     ui_todo("Now install {ui_value(cpp_register_deps[!installed])} to use cpp11.")
   }
 }
+
+get_cpp_dynlib <- function() {
+  desc <- desc::desc(package = NULL)
+  desc$get_list("Package")[[1]]
+}
+
+check_cpp_dynlib <- function() {
+  pkgname <- get_cpp_dynlib()
+  roxygen_ns_append(
+    sprintf("@useDynLib %s, .registration = TRUE", pkgname)
```

Comment:
You could use `project_name()` here instead of a custom function.

And then I think it's short enough that it would be clearer to inline directly in `use_cpp11()`.